test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+odoo-addon-mrp_flattened_bom_xlsx_direct_materials_cost@git+https://github.com/OCA/manufacture-reporting.git@refs/pull/116/head#subdirectory=setup/mrp_flattened_bom_xlsx_direct_materials_cost
+odoo-addon-mrp_flattened_bom_xlsx@git+https://github.com/OCA/manufacture-reporting.git@refs/pull/112/head#subdirectory=setup/mrp_flattened_bom_xlsx
+odoo-addon-report_xlsx@git+https://github.com/OCA/reporting-engine.git@16.0#subdirectory=setup/report_xlsx
```

Comment:
Hi @PauBForgeFlow 

as commented in others PRs, could you remove the test-requirements and rebase this PR? the dependency is merged.